src/pages/workspace/WorkspaceProfilePage.js

```diff
@@ -168,6 +173,15 @@ function WorkspaceProfilePage({policy, currencyList, route}) {
                                     </Text>
                                 </View>
                             </OfflineWithFeedback>
+                            {!readOnly && (
+                                <Button
+                                    style={styles.mt6}
+                                    innerStyles={[StyleUtils.getHeight(variables.workspaceShareButtonHeight), StyleUtils.getWidthStyle(variables.workspaceShareButtonWidth)]}
+                                    text={translate('common.share')}
+                                    onPress={onPressShare}
+                                    small
```

Comment:
Try changing this prop and you will notice we get the desired button size.

Reply:
@shawnborton updated

Reply:
Thanks! I think we just need updated screenshots now to check.

Reply:
<img width="709" alt="Screenshot 2024-02-22 at 16 00 07" src="https://github.com/Expensify/App/assets/28352309/0f7c8ea0-3409-49f6-96c3-3bd7182d092a">
<img width="359" alt="Screenshot 2024-02-22 at 16 00 30" src="https://github.com/Expensify/App/assets/28352309/8c6334fa-685a-4159-aee4-bc920a0e6d44">
@shawnborton here they are